Wiki.md

```diff
@@ -177,3 +177,12 @@ If the deployment of changes to the wiki following the merging of a pull request
 1. Navigate to the Oppia wiki repository's Actions tab.
 2. Select the "Deploy to wiki" workflow.
 3. Refer to these [instructions](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow?tool=webui#running-a-workflow) to manually execute the workflow.
+
+## Tips and Try
+
+* ### How can I see markdown (wiki) preview in Visual Studio Code?
+
+Right-click on the editor Tab and select **Open Preview** (Windows/Linux: `Ctrl+Shift+V`, Mac: `Command+Shift+V`) or use the **Command Palette** Windows/Linux: `Ctrl+Shift+V`, Mac: `Command+Shift+V` to run the **Markdown: Open Preview to the Side** command (Windows/Linux: `Ctrl+K V`, Mac: `Command+K V`). 
```

Comment:
Is it possible to use the `<kbd>` tag for keys?

Reply:
Yes it's possible, I'll add keboard symbols.

Reply:
@MohitGupta121 sry if I wasn't clear. I meant `<kbd>Shift</kbd>` for each key. This should render as <kbd>Shift</kbd>

Reply:
Done.